### PR TITLE
EVEREST-1765 | fix pre-delete script

### PR DESCRIPTION
**Pull request by mayankshah1607**
* Since we did not specify `--cascade=foreground` in `kubectl delete`, it defauls to `background` and ends up removing the `foregroundDeletion` finalizer from our objects, which results in a broken deletion.

### CODE REVIEW

**Comment by mayankshah1607** [comment]
CI is blocked by https://github.com/percona/percona-helm-charts/pull/451

**Review by recharte — APPROVED** [review]

**Comment by recharte** [comment]
Merged #449 to unblock this

**Pull request merged**
